WalletWasabi.Fluent/Controls/PreviewItem.axaml.cs

```diff
@@ -40,7 +40,11 @@ public PreviewItem()
 				if (obj.ToString() is { } text)
 				{
 					_copyButtonPressedStopwatch = Stopwatch.StartNew();
-					await Application.Current.Clipboard.SetTextAsync(text);
+
+					if (Application.Current is { Clipboard: { } })
```

Comment:
Please use `if (Application.Current is { Clipboard: { } clipboard })` for consistency and I think it' safer  way.